b2sdk/utils/typing.py

```diff
@@ -0,0 +1,15 @@
+######################################################################
+#
+# File: b2sdk/utils/typing.py
+#
+# Copyright 2023 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from typing import TypeVar
+
+from b2sdk.transfer.outbound.upload_source import AbstractUploadSource
+
+TypeUploadSource = TypeVar("TypeUploadSource", bound=AbstractUploadSource)
```

Comment:
should this become a part of the public interface of the library?

Reply:
most likely not, gonna prefix it with underscore

Reply:
done